spec/integration/driver_spec.rb

```diff
@@ -809,6 +809,22 @@ def create_screenshot(file, *args)
       expect(@driver.window_handles).to eq(['0', '1'])
     end
 
+    it 'inherits settings in new windows' do
+      @session.visit '/'
+
+      new_tab = @session.window_opened_by do
+        @session.execute_script <<-JS
+          window.open('/poltergeist/simple')
+        JS
+      end
+
+      @session.within_window(new_tab) do
+        expect(
+          @driver.evaluate_script('[window.innerWidth, window.innerHeight]')
+        ).to eq([1024, 768])
```

Comment:
Since the test is that it inherits from current window please compare to the current window size rather than hard coded size